frontend/ui/data/credefaults.lua

```diff
@@ -0,0 +1,19 @@
+return {
+    prefix = 'copt',
+    options = {
+        screen_mode = "portrait",
+        view_mode = 0,
+        render_dpi = 96,
```

Comment:
This is the existing default value? I'm sure it must all overwritten somewhere, but I believe 160 is the program's base.

Reply:
This is crengine own "render dpi", the one used to scale CSS pixels and images (https://github.com/koreader/crengine/pull/213). It should stay at 96 (1 css px = 1 screen px), because everything looks too big with a bigger value (https://github.com/koreader/crengine/pull/213#issuecomment-401588194). Users can still increase it or set it as the default if needed.

Reply:
I see. Yes, 1 CSS px is 1/96 inch.